omrmakefiles/rules.ztpf.mk

```diff
@@ -16,6 +16,7 @@
 #    Multiple authors (IBM Corp.) - initial implementation and documentation
 #    James Johnston (IBM Corp.) - initial z/TPF Port Updates
 #    James Johnston (IBM Corp.) - further remove non-z/TPF Platform definitions
+#    James Johnston (IBM Corp.) - remove ld ORIGIN specifiers
```

Comment:
Please don't add this line - the new guidelines on licensing indicate that we shouldn't be adding a contributors list.  Also, this is guaranteed to conflict with the relicensing PRs.